clouddriver-cloudfoundry/src/main/java/com/netflix/spinnaker/clouddriver/cloudfoundry/client/Organizations.java

```diff
@@ -70,4 +71,35 @@ public CloudFoundryOrganization findById(String orgId) throws CloudFoundryApiExc
           .build())
       );
   }
+
+  public Optional<CloudFoundrySpace> findSpaceByRegion(String region) {
+    CloudFoundrySpace space = CloudFoundrySpace.fromRegion(region);
+
+    // fully populates the space guid which is what Cloud Foundry's API expects as an input, not the name.
+    Optional<CloudFoundryOrganization> organization = findByName(space.getOrganization().getName());
+
+    Optional<CloudFoundrySpace> spaceOptional = organization
+      .map(org -> findSpaceByName(org, space.getName()));
+
+    spaceOptional.ifPresent(spaceCase -> {
+      if (!(space.getName().equals(spaceCase.getName()) &&
+        space.getOrganization().getName().equals(spaceCase.getOrganization().getName()))) {
+        throw new CloudFoundryApiException("Org or Space name not in correct case");
```

Comment:
Is this possible? Is CAPI's name filter case insensitive? (https://apidocs.cloudfoundry.org/7.9.0/organizations/list_all_organizations.html)

Reply:
Yes, the CAPI name filter is case-insensitive.